source/js/navigation.js

```diff
@@ -66,26 +66,11 @@ function toggle_menu() {
  */
 function check_resize() {
     // If in desktop viewport (width needs to be the same as in nav_style.css):
```

Comment:
this is actually very interesting. if I was to edit nav_style.css, how would I know to come here and change this as well?

Reply:
That's the beauty of this, you won't 🤣 .